tests/test_buffer.py

```diff
@@ -982,6 +982,16 @@ def test_abs(self):
         sarr = sarr.abs()
         self.assertEqual(sarr.sum(), True)
 
+    def test_argminmax(self):
```

Comment:
Please devise a couple of more fixtures and use at least one fixture of one-dimensional array.